Update dependencies

abeforgit opened the pull request:
Build is broken with infinite loop atm, opening as draft to start discussion
@benjay10 iirc you dealt with this before
Might be worth just bumping to ember 3.28 in one go

abeforgit committed:
WIP Update dependencies

Broken build with infinite loop

benjay10 commented:
No problems building here, but nothing is really working, probably due to divergent Ember packages. This was the next plugin were I was going to do an Ember upgrade, so now is the time.

Commit:
Ember-cli-update v3.24.0...v3.28.5

Commit:
Remove embroider test-setup

This package and its config caused infinite build times

Commit:
Linting → removal of more embroider tests

Linting pointed out more places Embroider should be removed, and also
pointed out an improvement on some test case.

Commit:
Reviewed ember-auto-import, and other packages

It could not find the services until auto-import was added as a
devDependency. Coincidence?

benjay10 commented:
Okay, so after some fiddling, it builds and runs again.
The infinite build times where caused by `@embroider/test-setup`. This is not the first project where this is encountered. The easy solution is of course to remove all related things, but there is probably be a better way to deal with Embroider.

Commit:
Renamed service left old file in app/

Thanks to @nvdk for reporting this.
